### PR TITLE
Pass through subcommands as if there were called directly

### PR DESCRIPTION
**Pull request by joedborg**
This is nearly all "automatic".  The only real issue is the `inspect` command; which will need fixing for strict anyway as `sudo` is not allowed.

### CODE REVIEW

File: snapcraft.yaml [hunk 1]
```diff
@@ -331,8 +333,8 @@ parts:
       echo "Creating inspect hook"
       cp $KUBE_SNAP_ROOT/scripts/inspect.sh .
 
-      # Add bash completion for microk8s.kubectl. 
-      ./kubectl completion bash | sed "s/complete -o default -F __start_kubectl kubectl/complete -o default -F __start_kubectl microk8s.kubectl/g" | sed "s/complete -o default -o nospace -F __start_kubectl kubectl/complete -o default -o nospace -F __start_kubectl kubectl/g" > kubectl.bash
+      # Add bash completion for microk8s kubectl.
+      ./kubectl completion bash | sed "s/complete -o default -F __start_kubectl kubectl/complete -o default -F __start_kubectl microk8s kubectl/g" | sed "s/complete -o default -o nospace -F __start_kubectl kubectl/complete -o default -o nospace -F __start_kubectl kubectl/g" > kubectl.bash
```
Comment (knkski) [inline snapcraft.yaml]: Is it worth keeping both of the completions? It seems useful to keep around for the snap users.

Comment (joedborg) [reply snapcraft.yaml]: Yeah I agree

**Comment by ktsakalozos** [comment]
@joedborg do you think we could add subcommand descriptions in the `microk8s --help`? We do something very similar in https://github.com/ubuntu/microk8s/blob/master/installer/common/definitions.py#L3

**Comment by joedborg** [comment]
Yeah, let me think how we can make those shared (this would also remove the need for the ls)

**Pull request merged**